src/pages/home/Home.tsx

```diff
@@ -9,6 +9,7 @@ import { useDispatch, useSelector } from "react-redux";
 import { getMoviesThunk } from "../../store/movies-store/getMovies.thunk";
 import { Store } from "../../store";
 import { Movie } from "../../api/movies-api/models/movie.model";
+import {search} from '../../store/movies-store/search.thunk';
 
 const Home = () => {
   const [modalActive, setModalActive] = useState(false);
```

Comment:
Please keep some structure in your variables definition. Just for example, first you define your local state, state selectors etc., then you define your functions, event handlers, then hooks ( useEffect for example )

P.S. Please check 30 line and you will understand what I am talking about :)